caffe2/perfkernels/adagrad_avx.cc

```diff
@@ -1,9 +1,13 @@
-#include "caffe2/perfkernels/adagrad.h"
+#include "caffe2/perfkernels/adagrad_avx.h"
 #include "caffe2/perfkernels/cvtsh_ss_bugfix.h"
 
 #include <emmintrin.h>
 #include <immintrin.h>
 
+#include <cstddef>
```

Comment:
Yes, you're absolutely right. We already need cstdint (which we don't include and we'll fix this) and just want to avoid additional header needed.

Reply:
Well...I think `<cstddef>` is there automatically as long as there's any pointer arithmetic because `NULL` and `ptrdiff_t` (result of pointer - pointer) is defined by it.

Reply:
Got it. Anyway, this is not a big deal so no need for change. My other comment is more important.